build/config.go

```diff
@@ -9,16 +11,23 @@ import (
 	"github.com/layer5io/meshery-consul/internal/config"
 
 	"github.com/layer5io/meshkit/utils"
+	"github.com/layer5io/meshkit/utils/kubernetes"
 	"github.com/layer5io/meshkit/utils/manifests"
 	smp "github.com/layer5io/service-mesh-performance/spec"
 )
 
+type Versions struct {
```

Comment:
Other adapters seem to be missing this struct. As we drive toward consistency across adapters, is there any reason that this struct is not standard?

Reply:
This is more like a utility struct for a very specific use case. This is not needed anywhere else.

Reply:
I politely and firmly reject this notion. Give me a ring.